examples/wasm/winit_wasm.rs

```diff
@@ -33,3 +44,58 @@ fn counter(mut state: Local<CounterState>, time: Res<Time>) {
 struct CounterState {
     count: u32,
 }
+
+#[derive(Default)]
+struct TrackInputState {
+    keys: EventReader<KeyboardInput>,
+    cursor: EventReader<CursorMoved>,
+    motion: EventReader<MouseMotion>,
+    mousebtn: EventReader<MouseButtonInput>,
+    scroll: EventReader<MouseWheel>,
+}
+
+fn track_input_events(
+    mut state: ResMut<TrackInputState>,
+    ev_keys: Res<Events<KeyboardInput>>,
+    ev_cursor: Res<Events<CursorMoved>>,
+    ev_motion: Res<Events<MouseMotion>>,
+    ev_mousebtn: Res<Events<MouseButtonInput>>,
+    ev_scroll: Res<Events<MouseWheel>>,
+) {
+    // Keyboard input
+    for ev in state.keys.iter(&ev_keys) {
+        if ev.state.is_pressed() {
+            log::info!("Just pressed key: {:?}", ev.key_code);
+        } else {
+            log::info!("Just released key: {:?}", ev.key_code);
+        }
+    }
+
+    // Absolute cursor position (in window coordinates)
+    for ev in state.cursor.iter(&ev_cursor) {
+        log::info!("Cursor at: {}", ev.position);
+    }
+
+    // Relative mouse motion
+    for ev in state.motion.iter(&ev_motion) {
+        log::info!("Mouse moved {} pixels", ev.delta);
+    }
+
+    // Mouse buttons
+    for ev in state.mousebtn.iter(&ev_mousebtn) {
+        if ev.state.is_pressed() {
+            log::info!("Just pressed mouse button: {:?}", ev.button);
+        } else {
+            log::info!("Just released mouse button: {:?}", ev.button);
```

Comment:
Would this track change - that is it prints "Just released mouse button" once? Or does it print this line for the whole program lifetime when mouse button isn't pressed?

Reply:
Once.
It is processing `Events<MouseButtonInput>` and these events are edge-triggered, not level-triggered.